core/src/io/anuke/mindustry/ui/dialogs/SettingsMenuDialog.java

```diff
@@ -252,6 +252,7 @@ public void add(SettingsTable table){
         });
         graphics.sliderPref("fpscap", 240, 15, 245, 5, s -> (s > 240 ? Core.bundle.get("setting.fpscap.none") : Core.bundle.format("setting.fpscap.text", s)));
         graphics.sliderPref("chatopacity", 100, 0, 100, 5, s -> s + "%");
+        graphics.sliderPref("lasersopacity", 50, 0, 100, 5, s -> s + "%");
```

Comment:
`graphics.checkPref("lasers", true);` is still in the settings, and `setting.lasers.name` needs to be removed from the main bundle as well.

Reply:
Forgot about that.

Reply:
Done